### ISSUE TITLE
Cannot find module './createMetatagsFromConfig' in webpack-pwa-manifest-plugin

### ISSUE THREAD

**Issue by Charlie-Hua** [expo/expo-cli#532]
## Description
Got `Cannot find module './createMetatagsFromConfig'` error in `webpack-pwa-manifest-plugin` when running `expo start --web --non-interactive`

## Expected Behavior

Browser opened without error.

### Error messages:
```
$ EXPO_DEBUG=true expo start --web --non-interactive 
[19:30:10] Expo DevTools is running at http://localhost:19002
[19:30:11] Cannot find module './createMetatagsFromConfig'
[19:30:11] Error: Cannot find module './createMetatagsFromConfig'
    at Function.Module._resolveFilename (module.js:548:15)
    at Function.Module._load (module.js:475:25)
    at Module.require (module.js:597:17)
    at require (internal/module.js:11:18)
    at _load_createMetatagsFromConfig (/Users/charlie/.config/yarn/global/node_modules/@expo/webpack-pwa-manifest-plugin/build/index.js:54:61)
    at new WebpackPwaManifest (/Users/charlie/.config/yarn/global/node_modules/@expo/webpack-pwa-manifest-plugin/build/index.js:124:51)
    at module.exports (/Users/charlie/.config/yarn/global/node_modules/@expo/webpack-config/webpack/webpack.common.js:166:5)
    at module.exports (/Users/charlie/.config/yarn/global/node_modules/@expo/webpack-config/webpack/webpack.dev.js:14:16)
    at module.exports (/Users/charlie/.config/yarn/global/node_modules/@expo/webpack-config/webpack.config.js:17:12)
    at Object.invokeWebpackConfig (/xdl@53.5.2/src/Web.js:28:10)
    at /xdl@53.5.2/src/Project.js:1859:20
    at Generator.next (<anonymous>)
    at step (/Users/charlie/.config/yarn/global/node_modules/xdl/build/Project.js:2365:191)
    at /Users/charlie/.config/yarn/global/node_modules/xdl/build/Project.js:2365:361
    at <anonymous>
```
### Files are missing in `/build`
![image](https://user-images.githubusercontent.com/361098/56423800-59c83480-62e0-11e9-9a39-d7448db9e293.png)

## Environment
```
  Expo CLI 2.15.1 environment info:
    System:
      OS: macOS 10.14.3
      Shell: 5.6.2 - /usr/local/bin/zsh
    Binaries:
      Node: 8.15.1 - /usr/local/bin/node
      Yarn: 1.13.0 - /usr/local/bin/yarn
      npm: 6.4.1 - /usr/local/bin/npm
      Watchman: 4.9.0 - /usr/local/bin/watchman
    IDEs:
      Android Studio: 3.0 AI-171.4443003
      Xcode: 10.2.1/10E1001 - /usr/bin/xcodebuild
    npmPackages:
      expo: ^33.0.0-alpha.web.1 => 33.0.0-alpha.web.1 
      react: 16.5.0 => 16.5.0 
      react-native: https://github.com/expo/react-native/archive/sdk-32.0.1.tar.gz => 0.57.1 
      react-navigation: 3.7.1 => 3.7.1 
    npmGlobalPackages:
      expo-cli: 2.14.0
```

**Comment by vstlouis** [comment]
Yep this started happening yesterday for me as well.

**Comment by fson** [comment]
Thanks for reporting. This is an issue with yesterday's release. The fix will be out soon.

**Comment by fson** [comment]
The issue has been fixed in `expo-cli@2.15.2`. Sorry for the trouble.
